package.json

```diff
@@ -11,6 +11,7 @@
     "url": "https://github.com/pkgjs/meet.git"
   },
   "scripts": {
+    "build": "npx @vercel/ncc build && npx convert-action",
```

Comment:
Would be great to install `convert-action` as a dev, and then convert the ziet version dev dep to vercel.  I know that is not required, but there are good reasons to be explicit about your deps.